recipes/draco/meta.yaml

```diff
@@ -0,0 +1,42 @@
+
+{% set name = "draco" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "9f369ce924974acf9ab582fe8a9e50d86d54346460b0ac6d3808e3fcf269489c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/draco/archive/{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - draco_encoder -h
+
+about:
+  home: https://github.com/google/draco/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE.txt
+  summary: 'A library for compressing and decompressing 3D geometric meshes and point clouds'
+
+  description: |
+    A library for compressing and decompressing 3D geometric meshes and point clouds
+  doc_url: https://google.github.io/draco/
+  dev_url: https://github.com/google/draco
+
+extra:
+  recipe-maintainers:
+    - hobu
+    - chambbj
```

Comment:
@chambbj Are you happy to be listed as a maintainer of this recipe?

Reply:
@chrisburr More than happy! Please proceed to merge.